app/views/requests/success.html.erb

```diff
@@ -1,40 +1,25 @@
 <div class='<%= dialog_column_class %> success-page'>
-  <%= render 'success_header' %>
-  <%= render 'searchworks_item_information' %>
-  <%= render 'shared/request_status_information' %>
-
-  <% if current_request.is_a?(MediatedPage) || current_request.symphony_response.success? %>
-    <dl class='dl-horizontal dl-invert'>
-      <dt class='sr-only'><%= current_request.class.human_attribute_name :user %></dt>
-      <% unless current_request.user.library_id_user? %>
-        <dd>
-          <span class="requested-by"><%= current_request.user.to_email_string %></span>
-        </dd>
-      <% end %>
+  <h1 id='dialogTitle'>
+    <span class="sul-i-check-2" aria-hidden="true"></span> We've received your request
```

Comment:
Yes, the span simply provides the icon and the text is to the right of the icon.